programs/lfx-mentorship/2025/02-Jun-Aug/README.md

```diff
@@ -1521,3 +1523,24 @@ CNCF - Cilium: Document technical outcomes on website (2025 Term 2)
     - Bill Mulligan(xmulligan, <bill.mulligan@isovalent.com>)
 - Upstream Issue: <https://github.com/cilium/cilium.io/issues/492>
 - LFX URL: https://mentorship.lfx.linuxfoundation.org/project/8c677ed1-cec0-44b7-93aa-60f90ddb7949
+
+### kube-burner
+
+#### Enhancements around k8s performance testing
+
+CNCF - kube-burner: Enhancements around k8s performance testing (2025 Term 2)
+
+- Description:
+  We intend to get some help around open issues in the repository and also come up with new use cases and scenarios for performance testing any kubernetes distribution. We love new perspectives and are always open to new ideas alongside what we have as tracked work in github issues.
+- Expected Outcome:
+  The knock down some of open critical issues and bring in new perspective to the project.
+- Recommended Skills:
+  - Golang
+  - Kubernetes
+  - Cloud Platforms
+- Mentor(s):
+  -  Vishnu Challa (@vishnuchalla, vchalla@redhat.com) 
+  -  Raul Sevilla (@rsevilla87, rsevilla@redhat.com)
+- Upstream Issues: https://github.com/kube-burner/kube-burner/issues 
```

Comment:
Do you have a specific upstream issue we can use here? It makes for a good place for candidates to reach out and discuss the projects.